src/components/composer/grid_composer/grid_composer.ts

```diff
@@ -91,15 +90,18 @@ export class GridComposer extends Component<Props, SpreadsheetChildEnv> {
   get containerStyle(): string {
     const isFormula = this.env.model.getters.getCurrentContent().startsWith("=");
     const cell = this.env.model.getters.getActiveCell();
-    let style: Style = {};
-    if (cell) {
-      const cellPosition = this.env.model.getters.getCellPosition(cell.id);
-      style = this.env.model.getters.getCellComputedStyle(
-        cellPosition.sheetId,
-        cellPosition.col,
-        cellPosition.row
-      );
-    }
+    const position = this.env.model.getters.getPosition();
+    const sheetId = this.env.model.getters.getActiveSheetId();
+    const cellPosition = this.env.model.getters.getMainCellPosition(
+      sheetId,
+      position.col,
+      position.row
+    );
```

Comment:
I'm not a great fan of this, we have to call 3 differents getters to have the position of the active cell...

Reply:
Yes I just realized this morning `getPosition` and `getMainCellPosition` should return a `CellPosition`. It's would be slightly better already.